appinventor/appengine/war/Ya.css

```diff
@@ -14,30 +14,61 @@ they get cut off in Windows.
 
 @font-face {
   font-family: Roboto;
-  src: url('/fonts/Roboto-Regular.ttf') format("truetype");
+  src: url('/fonts/Roboto-Regular.ttf');
 }
 
 @font-face {
   font-weight: bold;
   font-family: Roboto;
-  src: url('/fonts/Roboto-Bold.ttf') format("truetype");
+  src: url('/fonts/Roboto-Bold.ttf');
+}
+
+
+@font-face {
+  font-family: OpenDyslexic;
+  src: url('/fonts/OpenDyslexic-Regular.otf');
+}
+
+@font-face {
+  font-weight: bold;
+  font-family: OpenDyslexic;
+  src: url('/fonts/OpenDyslexic-Bold.otf');
 }
 
 body, html {
   height: 100%;
 }
 
+
+
+.gwt-bodyOdf{
+  font-family: OpenDyslexic, Helvetica, Arial, sans-serif;
+}
+
+.gwt-bodyRob{
+  font-family: Roboto, Helvetica, Arial, sans-serif;
+}
+
+
+.gwt-bodyRob .gwt-bodyOdf{
```

Comment:
If you swap the order of `.gwt-bodyRob` and `.gwt-bodyOdf` then you may not need this rule.